hack/generators/controllers/networking/main.go

```diff
@@ -725,8 +725,13 @@ func (r *{{.PackageAlias}}{{.Kind}}Reconciler) Reconcile(ctx context.Context, re
 		{{- end }}
 
 		{{- if .ProgrammedCondition.UpdatesEnabled }}
-		log.V(logging.DebugLevel).Info("Updating programmed condition status", "namespace", req.Namespace, "name", req.Name)
 		configurationStatus := r.DataplaneClient.KubernetesObjectConfigurationStatus(obj)
+		log.V(logging.DebugLevel).Info("Updating programmed condition status", "namespace", req.Namespace, "name", req.Name, "configuration_status",configurationStatus)
+		// REVIEW: Should we try to fetch the latest version of updated object to reduce probalility of race on update?
```

Comment:
I think we should be fine with just using the object fetched from the cache at the beginning of the reconciliation. As I understand, the issue we had was caused by concurrent writes with different statuses that made the "old" instances fight with the "new" instances. Now, if we skip updates to `Unknown` status from "new" instances, there will be no fight between them.

Reply:
Done.